From 353dfc3d3ea0ea209805210f4b0a6522bd815ce8 Mon Sep 17 00:00:00 2001
From: Lucas Salles <lucas.salles@policorp.com.br>
Date: Fri, 10 May 2024 10:59:52 -0300
Subject: [PATCH] Fix Positivo N14EPE and N15EPE key toggle touchpad and search
 key

(cherry picked from commit 6915e6d914fdae903e881d5e2ea4409cde5abcb9)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 60994f2a5d..3f2b820220 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -2121,6 +2121,11 @@ evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:svnPositivo*:pn*:pvr*:rvnPositi
  KEYBOARD_KEY_76=f21                                    # Fn+F1 toggle touchpad
  KEYBOARD_KEY_dd=search
 
+# Positivo (N15EPE, N14EPE)
+evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:svnPositivo*:pn*:pvr*:rvnPositivo*:rnN1[45]EPE*
+ KEYBOARD_KEY_76=f21                                    # Fn+F1 toggle touchpad
+ KEYBOARD_KEY_6e=search
+
 # Positivo (CG15D)
 evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:bvr*:svnPositivoTecnologiaSA:pn*:pvr*:rvn*:rnCG15D*
 # Positivo Motion (N14DP6, N14DP7, N14DP7-V2, N14DP9, N14JP6, N14KP6)
